### DISCUSSION TITLE
Unable to fork any repository

### DISCUSSION THREAD

**Discussion by mooleshacat** [community/community#145524] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am unable to fork any repositories... I was able to before but now when I click fork instead of forking it and taking me to my new fork, it just reloads the repository page.

I am not sure what happened, but the fork button does nothing.

I have tried with brave shields disabled and it still does not work

**Comment by mooleshacat** [comment, marked as answer]
It turns out my account was flagged for some weird reason.

This is why I cannot fork anything.

I am contacting GitHub because now all of my projects are on hold thanks to them flagging my account for no reason.

**Reply by Krishna23013541** [reply]
How did you fix this issue? I am facing the same issue for more than a week. Kindly guide me to fix this issue

**Comment by atanu2531** [comment]
I am also facing  same  issue  .. please  help  ..

**Comment by Pratishta08** [comment]
I am also facing the same issue, can u please tell me how to fix it
